package.json

```diff
@@ -57,6 +57,7 @@
         "@babel/preset-env": "^7.3.1",
         "@babel/preset-flow": "^7.0.0",
         "@babel/preset-react": "^7.0.0",
+        "@testing-library/react": "^10.0.1",
```

Comment:
I was trying to avoid adding another dependency and write tests using what we already had (Jest and React Test Renderer), but I find React Testing Library really useful and it's recommended by the React team:

> We recommend using React Testing Library which is designed to enable and encourage writing tests that use your components as the end users do.

Reply:
Yeah, it's a great tool. But I thought that what we were using is the same underlying lib.

Reply:
True! `@testing-library/react` creates helper functions for `react-dom/test-utils`. I think it makes tests easier to write and I like that the methods they expose force a certain testing philosophy. When I used `enzyme` (which is still great), by comparison, I felt like I was testing the implementation more than the functionality.